src/App.jsx

```diff
@@ -21,13 +21,87 @@ import CompletedItem from './components/CompletedItem';
 
 /*
   TODO:
-
   1.) Consider adding global css file with basic configuration in place
     - Check for comfort level changing styles
   2.) How do we determine the level of candidate?
     –
 */
 
 export default function App() {
-  return(<div></div>);
-}
+  const [toDosState, setToDosState] = useState([]);
+
+  const [completedState, setCompletedState] = useState([]);
+
+  const [idState, setIdState] = useState(1);
+
+  const updateToDo = (id) => {
+    const updatedToDos = [...toDosState].filter((todo) => todo.id !== id);
+    // will create new array where id is not equal to
+    // todo is the n th iteration in the array, will
+    // iterate every every element in the array
+    const updatedComplete = [...toDosState].filter((todo) => todo.id === id);
+
+    const singleComplete = {
+      ...updatedComplete[0],
+      completed: !updatedComplete[0].completed,
+    };
+    console.log(updatedComplete);
+    console.log(updatedToDos);
+    setToDosState(updatedToDos);
+
+    const TotalUpdatedComplete = [...completedState, singleComplete];
+    console.log(TotalUpdatedComplete);
+    setCompletedState(TotalUpdatedComplete);
+  };
+
+  const updateComplete = (id) => {
```

Comment:
This function is very similar to `updateTodo`.

I think you could accomplish the same thing with one function